src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -396,6 +397,7 @@ public abstract partial class HttpHeaders : System.Collections.Generic.IEnumerab
         public void Add(string name, string value) { }
         public void Clear() { }
         public bool Contains(string name) { throw null; }
+        internal virtual void EnsureHeaders() { }
```

Comment:
If it's an internal method, then you don't need to put this declaration into the 'ref' file.  So, delete this line change.